src/events/message.rs

```diff
@@ -106,6 +106,9 @@ async fn handle_highlighting(ctx: &client::Context, msg: &Message) -> Result<()>
             .footer(|f| f.text(format!("#{}", channel.name)));
 
         for user_id in users {
+            if user_id == &msg.author.id {
+                return Ok(());
+            }
```

Comment:
it should only `continue;`, not return - otherwise the other users don't get notified either